### ISSUE TITLE
Add support for BTECH DMR-6X2 Pro

### ISSUE THREAD

**Issue by hmatuschek** [hmatuschek/qdmr#260]
This device identifies itself as "D6X2UV2" with hardware revision V100 (1.00). 

This device is very likely almost identical to the D878UVII, so "only" the codeplug format needs to be verified:

- [x] Verify memory regions
- [x] Verify channel encoding
- [x] Verify zone encoding
- [x] Verify contact encoding
- [x] Verify analog contact encoding
- [x] Verify scan list encoding
- [x] Verify radio DMR ID encoding
- [x] Verify DMR APRS encoding
- [x] Verify general settings encoding
- [x] Verify boot settings encoding
- [x] Verify preset messages encoding
- [x] Verify hot key settings encoding
- [x] Verify FM broadcast channel encoding
- [x] Verfy analog signalling (DTMF, 5-tone etc) encoding
- [x] Verify DMR encryption encoding
- [x] Verify call-sign DB encoding.

Development will happen in the `d6x2uv2` branch. There is also an emulator script under `doc/reveng/btech/d6x2uv2/`, if you like to assist in the reverse engineering of the codeplug format.

**Comment by dbrianhendrix** [comment]
Please let me know what I can do to help!

**Comment by hmatuschek** [comment]
> Is that the same model as in #335 ?

Yes, it is. There is a different CPS for that device, so it might have some differences in the codeplug. I "just" need to verify the codeplug format first and then I can support it. However, as usual for AnyTone-type radios, it is a lot of work. So I would like to complete the support for the existing devices before I support new ones.

**Comment by EntropyWorks** [comment]
Keep up the great work!  
`git checkout origin/d6x2uv2`
Getting close!
`Cannot connect to radio: In radio.cc:70: Unhandled device BTECH 'DMR-6X2UV Pro'. Device known but not implemented yet.`
